GPU/GPUCommon.cpp

```diff
@@ -22,6 +24,12 @@ GPUCommon::GPUCommon() :
 	dumpNextFrame_(false),
 	dumpThisFrame_(false)
 {
+	DisplayList x;
+
+	// This assert failed on GCC x86 32-bit (but not MSVC 32-bit!) before adding the 
+	// "padding" field at the end. This is important for save state compatibility.
+	static_assert(sizeof(DisplayList) == 456, "Bad DisplayList size");
```

Comment:
Oh, I get it now, by 8 you meant 8 bytes not bits.  Nevermind, I didn't realize there was a 64 bit value.

-[Unknown]